assets/dockerfiles/fx/d/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM metrue/fx-d-base
```

Comment:
@andre2007 

I already pushed the image generated by your initial Dockerfile in assets/dockerfiles/base/d to Docker Hub already.